spec/requests/items_requests_spec.rb

```diff
@@ -204,7 +204,7 @@
         expect(response.body).to include('Category')
         expect(response.body).to include('CURRENTCATEGORY')
         expect(response.body).to include('Value Per Item')
-        expect(response.body).to include('20000')
+        expect(response.body).to include('$200.0')
```

Comment:
Dollar currency is always shown with two decimal places -- so I would expect this to be $200.00, not merely $200.0.